en/config/safety.md

```diff
@@ -0,0 +1,234 @@
+# Safety Configuration (Failsafes) 
+
+PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+
+The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+
+## Failsafe Actions {#failsafe_actions}
+
+Each failsafe defines its own set of actions. Some of the more common failsafe actions are:
+
+Action | Description
+--- | --- 
+None/Disabled | No action (the failsafe will be ignored).
+Warning | A warning message will be sent to *QGroundControl*.
+[Hold mode](../flight_modes/hold.md) | The vehicle will enter *Hold mode*. For multicopters this means the vehicle will hover, while for fixed/wing the vehicle will circle.
+[Return mode](../flight_modes/return.md) | The vehicle will enter *Return mode*. Return behaviour can be set in the [Return Home Settings](#return_settings) (below).
+[Land mode](../flight_modes/land.md) | The vehicle will enter *Land mode*, and lands immediately.
+RC Auto Recovery (CASA Outback Challenge rules) | TBD
+Terminate/Flight termination | Turns off all controllers and sets all PWM outputs to a failsafe value (defined in airframe configuration using `FAILSAFE` variable).
```

Comment:
Thanks for previous info - just copied in here.
But I'm not sure how this really differs from Lockdown. Presumably you could set failsafe to some value for motors to still spin (or whatever) but wouldn't this be pretty pointless for a quadcopter since you wouldn;t have any controllers running?
Upshot, in practise, how does it differ from Kill switch :-)